sdk/include/ndk/kefuncs.h

```diff
@@ -33,6 +33,53 @@ extern "C"
 
 #ifndef NTOS_MODE_USER
 
+//
+// Affinity helpers
+//
+FORCEINLINE KAFFINITY AFFINITY_MASK(ULONG Index)
```

Comment:
It _might_ be of interest to add minimal `_In_`, `_Out_` SAL2 annotations in the parameters, otherwise LGTM.